lib/cfg.c

```diff
@@ -524,27 +524,25 @@ cfg_dump_processed_config(GString *preprocess_output, gchar *output_filename)
     }
 }
 
-gboolean
-cfg_load_config(GlobalConfig *self, gchar *config_string, gboolean syntax_only, gchar *preprocess_into)
+static GString *
+_load_file_into_string(const gchar *fname)
 {
-  gint res;
-  CfgLexer *lexer;
-  GString *preprocess_output = g_string_sized_new(8192);
+  FILE  *fd = fopen(fname, "r");
 
-  lexer = cfg_lexer_new_buffer(self, config_string, strlen(config_string));
-  lexer->preprocess_output = preprocess_output;
+  if (!fd)
```

Comment:
It would be nice to log the reason here.

Reply:
The file is opened again where the reason not able to open a file is logged. I wanted to avoid logging the same thing twice (the two open's result could differ, but I think that has a low chanse, while this failing this open is not critical, while the other is.)